src/main/java/de/flapdoodle/embed/process/runtime/Starter.java

```diff
@@ -46,9 +46,15 @@ protected Starter(IRuntimeConfig config) {
 	}
 
 	public EXECUTABLE prepare(CONFIG config) {
-		
-		Distribution distribution = Distribution.detectFor(config.version());
-		
+		return prepare(config, null);
```

Comment:
Can you change it to: 
  return prepare(config, Distribution.detectFor(config.version()));